content/en/blog/_posts/XX-dra-consumable-capacity.md

```diff
@@ -0,0 +1,189 @@
+This enabled Kubernetes to model shareable hardware more accurately.
+
+But there was still a missing piece: it doesn't yet support scenarios
+where the device driver manages fine-grained, dynamic portions of a device resource—like network bandwidth—based on user demand,
+or to share those resources independently of ResourceClaims, which are restricted by their spec and namespace.
+
+That’s where the DRAConsumableCapacity feature comes in.
+
+## 🎯 What Does the DRAConsumableCapacity Feature Unlock?
+
+### Device sharing across multiple ResourceClaims or DeviceRequests
+
+With the DRAConsumableCapacity feature, resource drivers can now support sharing the same device—or even a slice of a device—across multiple ResourceClaims or across multiple DeviceRequests.
+
+This means that Pods from different namespaces can simultaneously share the same device.
+
+### Device resource allocation
+
+The DRAConsumableCapacity feature extends the allocation algorithm in the scheduler to support allocating a portion of a device's resources as defined in the `capacity` field,
+and to ensure that the total allocated capacity across all consumers **never exceeds the device’s total capacity**, even when shared across multiple ResourceClaims or DeviceRequests.
+
+This feature expands support for scenarios where the device driver is able to manage resources *within* a device on a per-process basis—for example,
+allocating a specific amount of memory (e.g., 8 GiB) from a virtual GPU
+or setting bandwidth limits on virtual network interfaces allocated to specific Pods. This aims to provide safe and efficient resource sharing.
+
+### DistinctAttribute constraint
+
+In addition to the existing `MatchAttribute` constraint, this feature introduces a new constraint: `DistinctAttribute`.
+
+The primary goal of `DistinctAttribute` is to **prevent the same device from being allocated multiple times within a single `ResourceClaim`**. This ensures that each allocation refers to a distinct resource, even if they belong to the same device class.
+
+It is useful for use cases such as allocating **network devices connecting to different subnets** to expand coverage or provide redundancy across failure domains.
+
+## 🌟 How to use?
+
+`DRAConsumableCapacity` is introduced as an alpha feature in Kubernetes 1.34. The [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) `DRAConsumableCapacity` must be enabled in kubelet, kube-apiserver, and kube-scheduler.
+
+```bash
+--feature-gates=...,DRAConsumableCapacity=true
+```
+
+### As a DRA driver developer
+
+As a DRA driver developer, you can make a `Device` within a `ResourceSlice` allocatable to multiple `ResourceClaims` or `DeviceRequests` by setting `AllowMultipleAllocations` to a pointer to the `true` value.
```

Comment:
```suggestion
As a DRA driver developer, you can make a `Device` within a `ResourceSlice` allocatable to multiple `ResourceClaims` or `DeviceRequests` by setting `AllowMultipleAllocations` to `true`.
```

I think that's semantically clearer (less distracting), and the code snippet still shows the `ptr` as it should.